pcweb/components/sidebar/sidebar.py

```diff
@@ -6,15 +6,22 @@
 from pcweb import styles
 from pcweb.components.navbar.state import NavbarState
 from pcweb.route import Route
-from pcweb.styles import font_weights as fw
 from .state import SidebarState, SidebarItem
-from .style import heading_style2, heading_style3
 
 from .sidebar_items.learn import learn, frontend, backend, hosting
 from .sidebar_items.component_lib import get_component_link, component_lib, other_libs
 from .sidebar_items.reference import api_reference, recipes, tutorials
 
 
+heading_style2 = {
+    "background_color": rx.color("violet", 3),
```

Comment:
Can we use "accent' instead of "violet" everywhere? Then we can theme the whole site together later if we want